src/StoreApi/Routes/CartUpdateCustomer.php

```diff
@@ -117,6 +117,15 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 				'shipping_country'    => isset( $shipping['country'] ) ? $shipping['country'] : null,
 			)
 		);
+
+		$shipping_phone_value = isset( $shipping['phone'] ) ? $shipping['phone'] : null;
+
+		if ( is_callable( [ wc()->customer, 'set_shipping_phone' ] ) ) {
+			wc()->customer->set_shipping_phone( $shipping_phone_value );
+		} else {
+			wc()->customer->update_meta_data( 'shipping_phone', $shipping_phone_value );
+		}
```

Comment:
Should we add a comment block here (todo) as a reminder to remove this when the core functionality is added in WC 5.6 (your PR in Woo Core got merged)?

_This comment also applies to other schema and route classes where the shipping phone value is set or retrieved_.

Reply:
Added in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4268/commits/9935cd1d1a4c78754cf49ab0c87673e27fc34619